dart-executer/dart-executer.py

```diff
@@ -0,0 +1,19 @@
+﻿from flask import Flask, request, jsonify
+import subprocess
+
+app = Flask(__name__)
+
+@app.route('/execute', methods=['POST'])
```

Comment:
![image](https://github.com/Scaleup-Excellenteam/dockers-flask-OriVered/assets/127769694/886708bd-211a-4c99-bf11-caed7c9ebc28)

Reply:
You should have not implemented it in python